app/assets/stylesheets/govuk_publishing_components/components/_subscription-links.scss

```diff
@@ -37,4 +46,15 @@
       background-size: 20px 14px;
     }
   }
+
+  .gem-c-subscription-links__feed-box {
+    padding: $gutter-half;
+    margin-bottom: $gutter-half;
+    background: $grey-3;
+  }
+
+  .gem-c-subscription-links__feed-description {
+    font-weight: bold;
```

Comment:
Should we be using our font mixins here?

Reply:
I think the only way to do that is to specify a font size as well? I deliberately didn't, because I didn't want to set a font size, only the weight.